orange/src/main/java/com/hartwig/hmftools/orange/conversion/OrangeConversion.java

```diff
@@ -154,13 +159,22 @@ public static PeachGenotype convert(@NotNull com.hartwig.hmftools.common.peach.P
                 .build();
     }
 
-    @NotNull
-    public static SignatureAllocation convert(@NotNull com.hartwig.hmftools.common.sigs.SignatureAllocation signatureAllocation)
+    @Nullable
+    public static List<SignatureAllocation> convertAndAnnotateWithEtiology(
+            @Nullable List<com.hartwig.hmftools.common.sigs.SignatureAllocation> signatureAllocation,
```

Comment:
Why can signatureAllocation be null?

Reply:
This input comes from `loadSigAllocations`, which could have a nullable output (this could e.g. happen when no sigs allocation tsv is provided)

Reply:
Clear! Didn't realise you changed the input from a single sig to a list of sigs, my bad!